jetty-io/src/main/java/org/eclipse/jetty/io/ByteBufferCallbackAccumulator.java

```diff
@@ -73,7 +73,7 @@ public int getLength()
 
     public void writeTo(ByteBuffer buffer)
```

Comment:
This param should be called `bufferInFillMode` to make it clear it is deviating from our standard policy of keeping buffers in flush state.  It also needs javadoc saying as much.

.... or better yet, do the flip to fill within this method so anybody calling it does not need to do the flip and flipback themselves

Reply:
I have implemented your second suggestion.